config/templates/cluster.yaml

```diff
@@ -475,6 +475,11 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 #     # See https://github.com/coreos/kube-aws/issues/230 for more information.
 #     rbac:
 #       enabled: true
+#     authentication:
```

Comment:
Write few lines commenting how to enable this auth or maybe just a link is enough